app/api/auth/agent-connect/callback/route.ts

```diff
@@ -62,6 +63,16 @@ const verifyAgentHabilitation = async (agent: IAgentInfo) => {
 
   const { isMCP } = agent;
 
+  // TEMPORARY
+  if (isMCP && !agent.siret) {
+    const { idpId } = agent;
+
+    if (idpId && idpToSiret[idpId]) {
+      agent.siret = idpToSiret[idpId];
+    }
+  }
+  // TEMPORARY
```

Comment:
I would rather have this at the same place whe define isMCP and other agent specific stuff as we might also use the siren in the log etc.